### ISSUE TITLE
error in org-ref-link-message

### ISSUE THREAD

**Issue by stevenbagley** [jkitchin/org-ref#156]
Place the following text on a line with the cursor right after the last right bracket:
`[[cite:author2016title]]`

This causes an error:
`Error running timer `org-ref-link-message': (error "Invalid search bound (wrong side of point)")`

**Comment by jkitchin** [comment]
i cannot reproduce this.

**Comment by stevenbagley** [comment]
I hope this helps. To reproduce: `emacs -Q`, then:
```
(require 'package)
(defvar emacs-d "/Users/sbagley/.emacs.d/")
(setq user-emacs-directory emacs-d)
(setq package-user-dir (expand-file-name "elpa" emacs-d))
(add-to-list 'load-path (expand-file-name "elisp" emacs-d))
(package-initialize)
(require 'org-ref)
```
then enter the one line as per initial issue report, and put cursor at end of line.

This is using: Mac OS X 10.11.3, Emacs 24.5.1, Org version 8.3.4

**Comment by jkitchin** [comment]
Yes, with org 8.3.4 I can reproduce it, but not with 8.2.10. I pushed a fix that I think works in 8.3.4 also. Apparently if you are the end of the line, you are "looking-at" "$", and not " ". Let me know if it helps.

**Comment by stevenbagley** [comment]
Yes, that resolves the error. Thanks.
